scripts/run.sh

```diff
@@ -1,16 +1,160 @@
 #!/bin/bash
```

Comment:
As a styleguide, can you follow a 2 spaces indentation? I noticed couple places use single space. Thanks :)